### ISSUE TITLE
How to use a static landing page?

### ISSUE THREAD

**Issue by kzenman** [givanz/VvvebJs#72]
I want to create a landing page with images for people to be able to select which theme they want to use, by clicking on a button. Adding the page is not a problem, how do I make it not editable? 
I tried the suggestion you gave some back in April but had no effect.
Tried in page header and body 
` body
    {
       --builder-filemanager-height: 0px;
    } 
`
Also tried in the editor.html head & body

Any help would be appreciated...TY!

**Comment by oleteacher** [comment]
Have you tried adding the class "data-vvveb-disabled" to the body or surrounding div?

**Comment by kzenman** [comment]
Sorry, I forgot to mention that...and yes it works, but disables the whole page and puts a "non-editable area" overlay.

**Comment by oleteacher** [comment]
"Sorry, I forgot to mention that..." Opps, I totally forgot that it does the overlay. Sorry could not be more help...

**Comment by kzenman** [comment]
For anyone else trying to achieve a non-editable landing page...here's how I did it (not sure if it's the best way). 
Changed editor.html to editor.php

in .../lib/builder/builder.js around line 750ish I changed 
`self._initHighlight();`
to 
`if(this.template === true){
 		self._initHighlight();
}`
Then in editor.php around line 935 after //run code after page/iframe is loaded
I added 
`      if('<?php echo $template; ?>' == '/sites/templates/home/index.html'){
        this.template = false;
      }else{
        this.template = true;
      }`

I forgot to mention I add this as a plugin to Codeigniter hence the echo $template which creates in new template page onload called /sites/templates/home/index.html

This way when the users want to edit an existing file, the editor will work as designed. 

Hope that helps...if someone has a easier way I'm always up for that. 

Best!
